draft-quic-atsss-reqs.mkd

```diff
@@ -127,8 +133,19 @@ This document briefly presents the Access Traffic Steering, Switching, and Split
 
 --- middle
 
+# Introduction {#intro}
+
+The 3rd Generation Partnership Project (3GPP) has described the Access Traffic Steering, Switching, and Splitting (ATSSS) service, used to direct TCP traffic between endpoints with multiple available paths, in their Release 16 {{TS23501}}. In Release 16, ATSSS uses the IETF MPTCP protocol {{RFC6824}}.
```

Comment:
> In Release 16, ATSSS uses the IETF MPTCP protocol {{RFC6824}}.

It also uses it in combination with the Convert Protocol, but that may be a detail that can be left out.

Reply:
I would also add the convert protocol since this is an IETF protocol developed in the tcpm working group.